src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -1940,6 +1955,7 @@ public partial class RankException : System.SystemException
     }
     public readonly ref partial struct ReadOnlySpan<T>
     {
+        private readonly object _dummy; //Manually set to object because tools don't know it is special
```

Comment:
@jkotas @VSadov should this be of type object or type T?

Reply:
`object` , there are no fields of type `T` in Span.

Reply:
It is similar to the `ImmutableArray<T>` - `T` is the element type, but the struct is just an indirection to some other storage, it does not contain any instances directly.

Reply:
I have the changed for ImmutableArray locally and my tool generates `internal T[] array;` for it as it is generic underneath.

Reply:
Either `T[]` or `object` would work since `T[]` is not a struct.

Reply:
Basically arrays are just reference types for this purpose.